utils/Formatter.tsx

```diff
@@ -297,8 +292,8 @@ export function getMinecraftColorCodedElement(text: string): JSX.Element {
         a: { fontWeight: 'normal', textDecoration: 'none', color: '#55ff55' },
         b: { fontWeight: 'normal', textDecoration: 'none', color: '#55ffff' },
         c: { fontWeight: 'normal', textDecoration: 'none', color: '#FF5555' },
-        e: { fontWeight: 'normal', textDecoration: 'none', color: '#FF55FF' },
-        d: { fontWeight: 'normal', textDecoration: 'none', color: '#FFFF55' },
+        d: { fontWeight: 'normal', textDecoration: 'none', color: '#FF55FF' },
+        e: { fontWeight: 'normal', textDecoration: 'none', color: '#FFFF55' },
```

Comment:
The color codes were reverted back to the old state (https://github.com/Coflnet/hypixel-react/pull/1053) because users reported the coloring to be wrong. I dont know if this is the only mixup, but the ones currently in develop seem to work.
I dont see a reason to not use them. (You can still order them if you want)